apps/coscrad-frontend/src/components/term-of-the-day/use-date.tsx

```diff
@@ -0,0 +1,7 @@
+export function useDate() {
+    const currentDay = new Date().getDate();
+
+    console.log('TODAYS DAY:', currentDay);
```

Comment:
You need to parse this down to a month \ day representation or maybe just a number (0-33) before returning. But otherwise, right on the money.